Ansible/inventories/inventory.ampr

```diff
@@ -0,0 +1,13 @@
+[ldap-servers]
+idp.se.ampr.org needs_certificate=false
```

Comment:
Hello @qasims - The IdP needs a certificate, so you should keep this "needs_certificate=true" otherwise, it looks ok and I can accept the PR :)